pkg/ocm/builder.go

```diff
@@ -30,8 +31,18 @@ func (ocb *ocmClientBuilder) New(c client.Client, ocmBaseUrl *url.URL) (OcmClien
 		return nil, fmt.Errorf("failed to retrieve cluster access token")
 	}
 
+	// Inherit environment proxy config
+	var proxyURL *url.URL
+	proxy := getProxy()
+	if proxy != "" {
+		proxyURL, err = url.Parse(proxy)
+		if err != nil {
+			return nil, fmt.Errorf("invalid-formatted proxy: %v", err)
+		}
+	}
+
 	// Set up the HTTP client using the token
-	httpClient := resty.New().SetTransport(&ocmRoundTripper{authorization: *accessToken})
+	httpClient := resty.New().SetTransport(&ocmRoundTripper{authorization: *accessToken, proxy: proxyURL})
```

Comment:
What's the value of `proxyURL` here if the `proxy = ""` above?

Reply:
It'll be the uninitialized pointer so will be nil. The roundtripper will not set the proxy if it's nil.